templates/fine-tune-llm_v2/cookbooks/modifying_hyperparameters/README.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Modifying hyperparameters\n",
+    "\n",
+    "**⏱️ Time to complete**: 10 minutes\n",
+    "\n",
+    "This guide will focus on how you can customize your fine-tuning run by modifying the various hyperparameters configurable. Make sure you've read the [basic fine-tuning guide](../../README.md) for better context. \n",
+    "\n",
+    "We provide a number of options to configure via the training YAML.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "## GPU resources\n",
+    "\n",
+    "Configuring GPU resources to be used is one of the most important pre-requisities for training. There are two fields in our YAML that are relevant:\n",
+    "\n",
+    "```yaml\n",
+    "num_devices: 16 # number of GPUs \n",
+    "worker_resources:\n",
+    "  accelerator_type:A10G: 0.001 # specifies GPU type available, and a minimum allocation per worker\n",
+    "```\n",
+    "\n",
+    "Internally, our fine-tuning code will launch Ray workers with each being allocated one GPU. The cluster will be auto-scaled if needed to meet the requirements. The different GPU types you can specify can depend on the specific Anyscale Cloud. The value you specify for the accelerator type does not matter much, as long as it's non-zero (so that each worker is allocated a GPU) and less than or equal to 1 (so that the requested number of GPUs is the same as `num_devices`)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "## Learning rate\n",
+    "\n",
+    "There are two entities of interest here: the actual learning rate value itself and the particular learning rate scheduler you use. The parameters you can control in the YAML are below: \n",
+    "\n",
+    "```yaml\n",
+    "learning_rate: 1e-4\n",
+    "lr_scheduler_type: cosine\n",
+    "num_warmup_steps: 10\n",
+    "```\n",
+    "\n",
+    "In the above config, the training run would use a cosine learning rate schedule (the default) with an initial warmup of 10 steps (the default). The peak learning rate would be 1e-4 (the value specified). \n",
+    "\n",
+    "We support both `'linear'` and `'cosine'` schedules. "
```

Comment:
in here take a screen shot of a cosine learning rate schedule so it quickly clicks to the user.